cmake/utils.cmake

```diff
@@ -21,10 +21,10 @@ function(shaderc_default_c_compile_options TARGET)
     endif()
     if (NOT SHADERC_ENABLE_SHARED_CRT)
       if (WIN32)
-	# For MinGW cross compile, statically link to the libgcc runtime.
-	# But it still depends on MSVCRT.dll.
-	set_target_properties(${TARGET} PROPERTIES
-		LINK_FLAGS "-static -static-libgcc")
+  # For MinGW cross compile, statically link to the libgcc runtime.
```

Comment:
Hmm. I was to fix the mixed indent characters earlier. It seems just didn't put enough spaces for tabs.